static/js/theme-map/Maps/Providers/Baidu.js

```diff
@@ -350,7 +350,8 @@ function load(resolve, reject, apiKey, {
   }
 
   const negativeLngFixStyle = document.createElement('style');
-  negativeLngFixStyle.innerText = negativeLngFixCSS;
+  // negativeLngFixStyle.innerText = negativeLngFixCSS;
+  negativeLngFixStyle.innerHTML = negativeLngFixCSS;
```

Comment:
is this to fix the HitchhikerJS issue? I think we'd need some other work around using innerHTML

Reply:
oops i left this in but i don't think this change affects the HitchhikerJS issue, it was just in InlineAssetHtmlPlugs